app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,12 +1,13 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
 import { Button, Pane } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Textarea, Select } from "neetoui/formik";
 
 import notesApi from "apis/notes";
 
-import { NOTES_FORM_VALIDATION_SCHEMA } from "../constants";
+import { NOTES_FORM_VALIDATION_SCHEMA, ROLES, TAGS } from "../constants";
```

Comment:
It seems to me you haven't updated the validation schema. If you don't update the validation schema how exactly are you going to get validations when interacting with the `Select` component etc?